src/Storefront/Resources/views/storefront/layout/navigation/offcanvas/navigation.html.twig

```diff
@@ -13,5 +13,8 @@
         </nav>
     {% endblock %}
 
-    {% sw_include '@Storefront/storefront/layout/navigation/offcanvas/categories.html.twig' with { navigation: navigation } %}
```

Comment:
not sure, if this would be a break :thinking:

Reply:
Yes it is, indeed. But without it, we cannot implement it because the whole behavior has changed. In my opinion, it is worth it because it fixes a lot of issues of the current off-canvas implementation, and the "break" is not that serious on the template level. Otherwise, we cannot backport these changes. 🤷‍♂️

Reply:
fine for me, I just wanted to point that out :slightly_smiling_face: